music-blocks.html

```diff
@@ -144,6 +144,34 @@ <h2>CREDITS</h2>
     </div>
 </section>
 
+<!-- "Refence Video" section -->
+<section id="testimonial" >
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12" >
+                <div class="row justify-content-center">
+                    <div class="col-md-8 col-md-push-2">
+                        <h2 class="text-center">REFERENCE VIDEOS</h2>
+                        <p class="customParagraphStyle">Explore more about Sugar labs and Music Blocks on our official youtube channel<a href="https://www.youtube.com/@SugarlabsOrg-EN" class="hrefCustomColor"> Sugar Labs</a>.</p>
+                        <div class="video-container">
+                            <div> 
+                                <iframe width="480" height="280"
+                                src="https://www.youtube.com/embed/actLKGJVa_Q">
+                                </iframe> 
+                            </div> 
+                            <div>  
+                                <iframe width="480" height="280"
+                                src="https://www.youtube.com/embed/RngzuKVl2Xk"> 
+                                </iframe>
+                            </div> 
+                        </div>  
```

Comment:
In addition to the two videos above, please also add:

https://www.youtube.com/watch?v=AZLQCDlKqdc

Reply:
So I will make these changes by this Sunday as I ll be traveling to my home for the next 2 days so will update you regarding this changes on Sunday